azure-pipelines-official.yml

```diff
@@ -135,27 +140,49 @@ extends:
         displayName: Official Build
         timeoutInMinutes: 360
         templateContext:
+          # This ensures that the SDL analysis runs only once and not for each output separately.
+          # https://eng.ms/docs/cloud-ai-platform/devdiv/one-engineering-system-1es/1es-docs/1es-pipeline-templates/features/outputs#multiple-outputs
+          outputParentDirectory: '$(Build.SourcesDirectory)/artifacts'
+
           outputs:
 
+          - output: pipelineArtifact
```

Comment:
These two outputs are just converted from below where they were defined as steps. They need to be defined as outputs so `outputParentDirectory` set above has an effect for them.